src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsstafflookupservice/integration/JwtAuthHelper.kt

```diff
@@ -55,11 +54,11 @@ class JwtAuthHelper {
       .also { scope?.let { scope -> it["scope"] = scope } }
       .let {
         Jwts.builder()
-          .setId(jwtId)
-          .setSubject(subject)
-          .addClaims(it.toMap())
-          .setExpiration(Date(System.currentTimeMillis() + expiryTime.toMillis()))
-          .signWith(keyPair.private, SignatureAlgorithm.RS256)
+          .id(jwtId)
+          .subject(subject)
+          .claims(it.toMap())
+          .expiration(Date(System.currentTimeMillis() + expiryTime.toMillis()))
+          .signWith(keyPair.private)
```

Comment:
as far as I can see the only change real change is removing `SignatureAlgorithm.RS256`. Is that because this is the default? Presumable we are not changing the signing algorithm.
I assume we have integration tests that would fail if this broke stuff?
Approving on above assumption

Reply:
looks like it is set by default now
https://javadoc.io/doc/io.jsonwebtoken/jjwt-api/0.10.6/io/jsonwebtoken/JwtBuilder.html#signWith-java.security.Key-